cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -181,7 +181,8 @@ public BuildCommand(Path userDir, PrintStream outStream, PrintStream errStream,
     @CommandLine.Option(names = "--code-coverage", description = "enable code coverage")
     private boolean coverage;
 
-    @CommandLine.Option(names = "--with-choreo", description = "package Choreo extension in the executable jar")
+    @CommandLine.Option(names = "--with-choreo", description = "package Choreo observability extension " +
```

Comment:
Shouldn't this and everywhere else be

```suggestion
    @CommandLine.Option(names = "--with-choreo", description = "package the Choreo observability extension " +
```

Reply:
+1

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5